lib/LaTeXML/Core/Stomach.pm

```diff
@@ -172,7 +172,7 @@ INVOKE:
       # Locally deactivate to avoid a flurry of errors in the same table.
       # Alert the end user once per table, allowing longer documents to not
       # hit 100 errors too quickly.
-      $STATE->assignMeaning(T_ALIGN,
+      $STATE->assignMeaning($token,
```

Comment:
Ah, and this bit - in this article a custom macro was `\let` to `&`, so the guard on `T_ALIGN` wasn't catching that custom macro. By using the exact token which snuck into the Stomach error, this should quiet a few further errors.